src/components/ConsoleLogComponent.tsx

```diff
@@ -0,0 +1,25 @@
+import config from './.config'; // Relative path to your File
+
+/**
+ /
+ * used to read console log configuration from a file to control
+ * what is need to be printed on console log
+ */
+export default class ConsoleLogComponent {
```

Comment:
Why is this even a class if it's only going to have one function? Can we make the method static in the class at least so we don't need to instantiate the object for no reason? It doesn't store any state.

Reply:
Now it is static